variables.tf

```diff
@@ -481,3 +481,9 @@ variable "docker_with_ssh_agent" {
   type        = bool
   default     = false
 }
+
+variable "docker_pip_cache" {
+  description = "Whether to mount a shared pip cache folder into docker environment or not"
+  type    = any
```

Comment:
`type = string` is better, right?

Reply:
It's not, the variable may be used in two ways:
* docker_pip_cache = true - creates a cache folder inside of $artifacts_dir/cache/pip
* docker_pip_cache = "/tmp/some_folder_path" - it's possibility to put the cache folder somewhere outside of $artifacts_dir and don't have an issue with incorrect permissions due to the folder usage inside of a docker container which runs by default with a root user permissions.

Reply:
Right, makes sense.